lms/templates/instructor/edx_ace/allowedunenroll/email/body.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% blocktrans %}You have been un-enrolled from {{ course_name }}{% endblocktrans %}
+            </h1>
+
+            <p style="color: rgba(0,0,0,.75);">
+                {% blocktrans %}You have been un-enrolled from course {{ course_name }} by a member of the course staff. Please disregard the invitation previously sent.{% endblocktrans %}
```

Comment:
un-enrolled -> unenrolled
from course -> from the course

Honestly not sure about "previously sent" but it's probably fine.